config/manager/manager-service.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: data-science-pipelines-operator-controller-manager
```

Comment:
can you shorten the name to `data-science-pipelines-operator`

Reply:
Done, updated the service name.